src/index.js

```diff
@@ -29,33 +33,41 @@ const client = new Client({
     StageInstanceManager: 0,
     VoiceStateManager: 0,
   }),
+  sweepers: {
+    messages: {
+      interval: 60000 * 5,
+      lifetime: 60000 * 15,
+    },
+  },
 });
 client.commands = new Collection();
-client.config = require('./config');
+client.cooldowns = new Collection();
 require('dotenv').config();
 
-const http = require('http');
 const server = http.createServer((req, res) => {
   res.writeHead(200).end('Express server is running.');
 });
+
 server.listen(3000, () => {
   console.log('Express server is ready.');
 });
 
-fs.readdirSync('./src/events/').forEach((file) => {
+readdirSync('./src/events/').forEach((file) => {
   const event = require(`../src/events/${file}`);
-  let eventName = file.split('.')[0];
-  console.log(`[ Event ] Loaded ${eventName}`);
-  client.on(eventName, event.bind(null, client));
+  if (event.once) {
+    client.once(event.name, (...args) => event.execute(...args));
+  } else {
```

Comment:
Codacy has a fix for the issue: [Closing curly brace appears on the same line as the subsequent block.](https://app.codacy.com/gh/Shadeslayer10/PokeRealm/pullRequest?prid=9275083)
```suggestion
  }
```